File: app/src/main/java/com/example/myapplication/tuan6/Product.java
package com.example.myapplication.tuan6;

public class Product {
    private String styleID;
    private String brand;
    private String price;
    private String additionalInfo;
    private String searchImage;

    public Product() {
    }

    public Product(String styleID, String brand, String price, String additionalInfo, String searchImage) {
        this.styleID = styleID;
        this.brand = brand;
        this.price = price;
        this.additionalInfo = additionalInfo;
        this.searchImage = searchImage;
    }

    public String getStyleID() {
        return styleID;
    }

    public void setStyleID(String styleID) {
        this.styleID = styleID;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

    public String getAdditionalInfo() {
        return additionalInfo;
    }

    public void setAdditionalInfo(String additionalInfo) {
        this.additionalInfo = additionalInfo;
    }

    public String getSearchImage() {
        return searchImage;
    }

    public void setSearchImage(String searchImage) {
        this.searchImage = searchImage;
    }
}

File: app/src/main/java/com/example/myapplication/tuan61/Product.kt
package com.example.myapplication.tuan6

class Product {
    var styleid : String ? = null;
    var brand : String ? = null;
    var price : Int ? = null;
    var product_info : String ? = null;
    var image : String ?= null;

    constructor(
        styleid: String?,
        brand: String?,
        price: Int?,
        product_info: String?,
        image: String?
    ) {
        this.styleid = styleid
        this.brand = brand
        this.price = price
        this.product_info = product_info
        this.image = image
    }

}